android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt

```diff
@@ -260,11 +307,11 @@ class GeoWidgetFragment : Fragment() {
         mapView.onSaveInstanceState(outState)
     }
 
-    fun addLocationToMap(location: GeoWidgetLocation) {
+    fun addLocationToMap(location: Feature) {
```

Comment:
Is this single-line function called in multiple places? If not I'd move the single code block to the call site to improve readability.